docs/oidc_testing/README.md

```diff
@@ -0,0 +1,81 @@
+# Bring up a cluster w/ the OIDC code enabled.
+1. Set up dev environment (which you've probably already done)
+   ```
+   git clone git@github.com:kubic-project/automation
+   cd automation
+   ./caasp-devenv --setup
+   ```
+1. check out OIDC-aware velum
+   ```
+   cd ../velum
+   git remote add dannysauer git@github.com:dannysauer/velum
```

Comment:
I don't think that readme in kubic-project should refer to personal github accounts. Especially not to one that is a copy of velum from within the velum repo itself.

Reply:
Correct.  Those two references need to be removed, since they won't be relevant after this is merged.  Thanks. :)

Fixed in cbed0d6